installers/ad-synchrona-14-leds.sh

```diff
@@ -55,6 +55,9 @@ do
 		LED_STATUS=$STATUS_RED
 	fi
 
+	echo "out" > "/sys/class/gpio/gpio$RED_LED1/direction"
+	echo "out" > "/sys/class/gpio/gpio$GREEN_LED1/direction"
```

Comment:
Yeah, I saw a similar thing on the main service. While in there it's more odd to me what could have been the issue/race, in here I can see what could be the race... When we reload (remove + aload) the overlay and the ad9545 device is unbind, the pinctrl settings are also removed which brings the pins back to their default (which might be input). If you happen to try to set a value at that moment, the issue pops up...

So yeah, for now let's have this workaround and keep things simple but I'm not sure if we would need some kind of synchronization between this script and the reload_dtb.sh